pkg/profiles/openapi.go

```diff
@@ -104,11 +108,18 @@ func swaggerToServiceProfile(swagger spec.Swagger, namespace, name, clusterDomai
 	return profile
 }
 
-func mkRouteSpec(path, pathRegex string, method string, responses *spec.Responses) *sp.RouteSpec {
+func mkRouteSpec(path, pathRegex string, method string, operation *spec.Operation) *sp.RouteSpec {
+	retryable := false
+	var responses = (*spec.Responses)(nil)
```

Comment:
This can simply be `var responses *spec.Responses`.  Declaring a variable will automatically initialize it with the zero value (nil).

Reply:
@adleong Thank you, this seems like a better way.:+1: I have made the change